.semaphore/semaphore.yml

```diff
@@ -73,7 +73,20 @@ blocks:
             - sem-version java $JAVA_VERSION
             - ./gradlew clean test
 
-  - name: Run first block of tests
+  - name: Run first block of tests (recipes)
+    execution_time_limit:
+      minutes: 10
+    task:
+      prologue:
+        commands:
+          - sudo apt-get install gradle-6.8.3 -y
+          - sudo update-alternatives --set gradle /usr/lib/gradle/6.8.3/bin/gradle
```

Comment:
Do we need Gradle for the recipes? It doesn't hurt anything though - I'll leave it up to you

Reply:
I left in there to future-proof in case we started moving test cases around and weren't sure why they were failing, but for cleanliness, I can remove.  WDYT?

Reply:
Yeah maybe just leave it for now